lib/helper/Puppeteer.js

```diff
@@ -59,7 +59,8 @@ const consoleLogStore = new Console();
  *
  * ```js
  * "chrome": {
- *   "executablePath" : "/path/to/Chrome"
+ *   "executablePath" : "/path/to/Chrome",
+ *   "browserWSEndpoint": "ws://localhost:3000"
```

Comment:
thanks. Can you add here an example here for using CodeceptJS with browserless.io?